src/components/20-molecules/datepicker/index.js

```diff
@@ -575,8 +561,8 @@ class AXADatepicker extends NoShadowDOM {
     this.error = null;
     if (isValid) {
       this._date = validDate;
-      this.initDate(validDate);
       this.outputdate = this.formatDate(validDate);
+      this.initDate(validDate);
```

Comment:
Why did you moved that 2 lines down?

Reply:
initdate needs to be called after the 'outputdate' property has been filled, because this property indicates that the user interacted with the component already.